### DISCUSSION TITLE
How to specify <html lang=""> for each directory in the App Router of Next.js 14.

### DISCUSSION THREAD

**Discussion by ks-ynu** [vercel/next.js#62044] [App Router]
I am a beginner in Next.js.
Currently, I am creating a website that supports both English and Japanese.
I achieve this by organizing them into directories such as example.com/ja and example.com/en.

However, in the default setup of Next.js, layout.js has <html lang="en"> written, and this is applied to Japanese pages as well.
Is there a recommended method to change the lang attribute value for each directory?

If possible, I would like to accomplish this in a straightforward manner without using dynamic routing as much as possible.

**Comment by mutasim77** [comment]
hey, check this discussion : https://github.com/vercel/next.js/discussions/49415
